DSCResources/MSFT_xFirewall/MSFT_xFirewall.Schema.mof

```diff
@@ -15,4 +15,4 @@ class MSFT_xFirewall : OMI_BaseResource
   [Write, Description("Documentation for the Rule")] String Description;
   [Write, Description("Path and file name of the program for which the rule is applied")] String ApplicationPath;
   [Write, Description("Specifies the short name of a Windows service to which the firewall rule applies")] String Service;
-};
+};
```

Comment:
Please end a file with a newline [See item 5 in style guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md)